assets/ui/environmentInfoScreen.ui

```diff
@@ -0,0 +1,58 @@
+{
+    "type": "EnvironmentInfoScreen",
+    "contents": {
+        "type": "relativeLayout",
+        "contents": [
+            {
+                "type": "UIBox",
+                "layoutInfo": {
+                    "width": 300,
+                    "height": 300,
+                    "position-horizontal-center": {},
+                    "position-vertical-center": {}
+                },
+                "content": {
+                	"type" : "relativeLayout",
+					"contents": [
+					    {
+					        "type": "UILabel",
+					        "id": "environmentInfoLabel",
+					        "text": "Environment Info",
+					        "layoutInfo": {
+					            "use-content-width": true,
+					            "position-horizontal-center": {},
+					            "height": 30
+					        }
+					    },
+					    {
+					        "type": "UIText",
+					        "text": "",
```

Comment:
This has no placeholder text and shows up as empty on startup. How about replacing it with "Hello there! 0 milliseconds have passed since you first opened this screen."? You can do this in the *.ui file as well as the Java class.